test/components/entity/basic-details.spec.js

```diff
@@ -152,7 +152,7 @@ describe('EntityBasicDetails', () => {
           return testData.standardEntities.last();
         })
         .respondWithData(() => testData.extendedAudits.sorted())
-        .respondWithData(() => [])
+        .respondWithData(() => testData.extendedEntityVersions.sorted())
```

Comment:
This should have been changed in #880. For #880, I searched for the word "diff", but since this was just an empty array, I didn't notice it. I caught it now because it was causing a problem: if the entity versions aren't returned, `scrollData()` isn't able to access `entityVersion` and throws an error.